git-archive-all

```diff
@@ -422,14 +460,21 @@ class GitArchiver(object):
 if __name__ == '__main__':
     from optparse import OptionParser
 
-    parser = OptionParser(usage="usage: %prog [-v] [--prefix PREFIX] [--no-exclude] [--force-submodules] [--extra EXTRA1 [EXTRA2]] [--dry-run] OUTPUT_FILE",
-                          version="%prog {}".format(__version__))
+    parser = OptionParser(
+        usage="usage: %prog [-v] [--prefix PREFIX] [--no-exclude] [--force-submodules]"
+              "       [--extra EXTRA1 [EXTRA2]] [--dry-run] OUTPUT_FILE",
+        version="%prog {0}".format(__version__)
+    )
 
     parser.add_option('--prefix',
                       type='string',
                       dest='prefix',
                       default=None,
-                      help="prepend PREFIX to each filename in the archive. OUTPUT_FILE name is used by default to avoid tarbomb. You can set it to '' in order to explicitly request tarbomb")
+                      help="""
```

Comment:
@asmodehn, I don't think this should not be a multiline string. It screws up the help output. It would be better to use a normal string like it was. Normal strings be broken up into multiple lines if line length is an issue.

```
  --prefix=PREFIX                                prepend PREFIX to each
                      filename in the archive.
                      OUTPUT_FILE name is used by default to avoid tarbomb.
                      You can set it to '' in order to explicitly request
                      tarbomb
```